src/main/kotlin/no/nav/familie/ef/sak/oppgave/OppgaveService.kt

```diff
@@ -32,7 +32,7 @@ class OppgaveService(private val oppgaveClient: OppgaveClient,
                      @Value("\${FRONTEND_OPPGAVE_URL}") private val frontendOppgaveUrl: URI) {
 
     private val logger = LoggerFactory.getLogger(javaClass)
-    val ENHET_NAY = "4489"
+    private val enhetNay = "4489"
```

Comment:
Dette hjører ikke hjemme i denne PR. 
Hvis denne skal endres burde den vel kanskje vært endret til en const i stedet?

Reply:
Det kunne vært const i stedet. Jeg har alltid vært av den mening av at man fikser feil og advarsler når man jobber i en fil.

Reply:
Tenker det er fint, men kanskje når man endrer en begrenset mengde med filer? :)

Reply:
Tenker en endring av aggregat BARE burde fikse aggregat, ikke for mye annet.
Og burde kanskje vært oppdelt i flere PR's:
* Aggregat, uthenting av aggregat - Dvs domeneklass, endring i repository (og metode i behandlingsservice?)
* Ta i bruk aggregat
* Ev sletting av unødvendig kode

Det gjør det mye enklere for diskusjon, før man har lagt ned en hel arbeidsdag